src/raster/r.random.walk/r.random.walk.html

```diff
@@ -0,0 +1,40 @@
+<h2>DESCRIPTION</h2>
+The <em>r.random.walk</em> module generates a 2D random walk across the current computational region.
+The module provides control of the number of steps and directions (4 or 8) a walker can take and allows 
+the walker's behavior to be set to be self-avoiding (Madras et al., 1996) or allow revisits. The output displays
+the frequency the walker visited each cell or the average frequency. The module can run multiple walks in
+parallel. It either samples the same starting location for each walk or generates a unique starting position
+for each walker.
+
+<h2>EXAMPLE</h2>
+Using the North Carolina full sample dataset:
+
+
+<div class="code"><pre>
+# set computational region
+g.region raster=elevation -p
+
+# calculate smoothed random walk from a single starting locations.
+r.random.walk -at output=random_walk_smooth_paths directions=8 steps=100000 memory=1800 seed=1 nprocs=6 nwalkers=100
+</pre></div>
+<center>
+    <img src="r_random_walk_example.png" width=600 border=0><br>
+    Smoothed random walk (Single Starting Location)
+</center>
+
+<div class="code"><pre>
+# calculate smoothed random walk from a multiple starting locations.
+r.random.walk -as output=random_walk_smooth directions=8 steps=100000 memory=1800 nprocs=6 nwalkers=100
+</pre></div>
+
+<center>
+    <img src="r_random_walk_example_2.png" width=600 border=0><br>
+    Smoothed random walk (Multiple Starting Locations)
+</center>
+
+<h2>SEE ALSO</h2>
+<a href="https://grass.osgeo.org/grass-stable/manuals/r.surf.fractal.html">r.surf.fractal</a>,
+<a href="https://grass.osgeo.org/grass-stable/manuals/r.surf.random.html">r.surf.random</a>
+
+<h2>AUTHOR</h2>
+Corey T. White, <a href="https://geospatial.ncsu.edu/geoforall/">NCSU GeoForAll Lab</a>
```

Comment:
Just a minor detail, good practice to end file with newline (see eg. https://stackoverflow.com/a/729795):
```suggestion
Corey T. White, <a href="https://geospatial.ncsu.edu/geoforall/">NCSU GeoForAll Lab</a>

```